examples/counter_in/cnt_dig_buff_cont_ext_clk.py

```diff
@@ -0,0 +1,35 @@
+﻿"""Example of counter input edge count operation.
+
+This example demonstrates how to count buffered digital events
+on a Counter Input Channel. The Initial Count, Count Direction,
+Edge, and Sample Clock Source are all configurable.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, CountDirection, Edge
+
+with nidaqmx.Task() as task:
+    task.ci_channels.add_ci_count_edges_chan(
+        "Dev1/ctr0",
+        edge=Edge.RISING,
+        initial_count=0,
+        count_direction=CountDirection.COUNT_UP,
+    )
+    task.timing.cfg_samp_clk_timing(
+        1000, source="/Dev1/PFI8", sample_mode=AcquisitionType.CONTINUOUS
```

Comment:
PFI8 is the default source terminal for ctr0, which is used for edge counting. Use PFI9 here, which is the default gate terminal, which is the sample clock pin

```suggestion
        1000, source="/Dev1/PFI9", sample_mode=AcquisitionType.CONTINUOUS
```

Reply:
FYI defaults can be found [here](https://www.ni.com/docs/en-US/bundle/daqhelp/page/xseriescountsigconn.html)